### PR TITLE
Review README contents

### PR DESCRIPTION
**Pull request by cristianonicolai**

### CODE REVIEW

File: packages/management-console/README.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
+#Management Console
```
Comment (nmirasch) [inline packages/management-console/README.md]: @cristianonicolai I think you missed an space between # and title

File: packages/management-console/README.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+##Local Development using mock backend server
```
Comment (nmirasch) [inline packages/management-console/README.md]: missing space here too

File: packages/management-console/README.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+This will start the frontend and backend using a mock server. The GraphiQL UI for the mock server will be available on http://localhost:4000/graphql
+if you would like to explore the data.  
+
+```bash
+yarn run dev
+```
+
+##Local Development using [Data Index](https://github.com/kiegroup/kogito-runtimes/wiki/Data-Index-Service) server
```
Comment (nmirasch) [inline packages/management-console/README.md]: and here

File: packages/management-console/README.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+
+Instead of starting a mock backend, you can connect to a running Data Index server which by default will be running on
+http://localhost:8180.
+
+```bash
+yarn run dev-remote-dataindex
+```
+
+##Executing tests
```
Comment (nmirasch) [inline packages/management-console/README.md]: missing space :)

File: packages/management-console/README.md [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+
+```bash
+yarn run test
+```
+
+##GraphQL Codegen
```
Comment (nmirasch) [inline packages/management-console/README.md]: and the last one

Comment (cristianonicolai) [reply packages/management-console/README.md]: @nmirasch thanks, should all be fixed now :)

**Review by AjayJagan — APPROVED** [review]
cool :+1:

File: packages/management-console/README.md [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+
+### Starting and Configuring the Keycloak Server
+
+To start a Keycloak Server you can use Docker and just run the following command:
+
+```
+docker run -e KEYCLOAK_USER=admin -e KEYCLOAK_PASSWORD=admin  -e KEYCLOAK_IMPORT=/tmp/kogito-realm.json -v <kogito-apps_absolutepath>/config/kogito-realm.json:/tmp/kogito-realm.json -p 8280:8080  jboss/keycloak
+```
+
+You should be able to access your Keycloak Server at [localhost:8280/auth](http://localhost:8280).
```
Comment (nmirasch) [inline packages/management-console/README.md]: just a minor thing, to be consistent with the other displayed urls in the page, Should we put here [http://localhost:8280/auth] displaying the http:// ?

**Review by nmirasch — APPROVED** [review]
I just added a minor comment for your consideration but this looks good to me!

File: packages/management-console/README.md (hunk 6 above)
Comment (cristianonicolai) [reply packages/management-console/README.md]: fixed, thanks :)

**Pull request merged**